.travis.yml

```diff
@@ -28,8 +28,15 @@ rvm:
   - 2.3.0
   - ruby-head
 matrix:
+  include:
+    - rvm: jruby-9.0.4.0 # Latest compiled version in http://rubies.travis-ci.org
+      jdk: oraclejdk8
+      env:
+        - "JRUBY_OPTS='-Xcompat.version=2.0 --server -Xcompile.invokedynamic=false'"
```

Comment:
per https://github.com/rails/rails/pull/16613 https://github.com/rails/rails/pull/17088 this should probably be `-J-Xmx1024M -Xcompat.version=2.0 --dev` but I'd want to verify

 cc @headius @robin850